lambda/readPass/index.js

```diff
@@ -84,47 +84,43 @@ exports.handler = async (event, context) => {
       }
       // Filter reservation number
       if (event.queryStringParameters.reservationNumber) {
-        queryObj.ExpressionAttributeNames['#registrationNumber'] = 'registrationNumber';
-        queryObj.ExpressionAttributeValues[':registrationNumber'] = AWS.DynamoDB.Converter.input(
+        queryObj.ExpressionAttributeValues[':reservationNumber'] = AWS.DynamoDB.Converter.input(
           event.queryStringParameters.reservationNumber
         );
         queryObj.FilterExpression += expressionBuilder(
           'AND',
           queryObj.FilterExpression,
-          '#registrationNumber =:registrationNumber'
+          'registrationNumber =:reservationNumber'
```

Comment:
`registrationNumber =:reservationNumber` shouldn't this be `registrationNumber =:registrationNumber `

Reply:
The column is named incorrectly in the database. It's really a `reservationNumber` not a `registrationNumber`.  There should probably be a tech-debt ticket created to rename it later and migrate the data.

Reply:
Good idea.

Reply:
Either way it's inconsistent.  It just depends if we want the inconsistency on lines 87+88, or on line 93.  But I can move the inconsistency back to 87+88 since that's where it was originally. 
I have created a ticket BRS-748 to address the tech-debt.

Reply:
Yeah originally I was just highlighting the #reservedWord change, so I was surprised to see name change come in afterward. I'd prefer to leave it as it was before the nit and address it in the ticket you just created.